google-cloud-firestore/src/main/java/com/google/cloud/firestore/BatchWriteResult.java

```diff
@@ -26,21 +26,18 @@
  */
 @InternalApi
 public final class BatchWriteResult {
-  private final DocumentReference documentReference;
+  private final String key;
```

Comment:
It looks like `BatchWriteResult` could be package-private, which would avoid the CLIRR violation.

Reply:
good point, done.

Reply:
It added another CLIRR violation for changing to package private, but I changed the rule.